server/routes/approved-dates/select-dates/selectApprovedDatesController.ts

```diff
@@ -37,12 +37,19 @@ export default class SelectApprovedDatesController implements Controller {
       }
     })
 
+    let backLink: string
```

Comment:
I think ternary fits better here, instead of a `let` you can use a `const`
```
const backLink = journey.datesToSave.length === 0
  ? ApprovedDatesUrls.reviewCalculatedDates(nomsId, journeyId)
  : ApprovedDatesUrls.reviewApprovedDates(nomsId, journeyId)
```